velox/functions/iceberg/Murmur3_32HashFunction.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <folly/CPortability.h>
+#include <cstdint>
+
+namespace facebook::velox::functions::iceberg {
+class Murmur3_32HashFunction {
```

Comment:
This function will also be used in Iceberg write, can we move it to a more common place such as `velox/common/iceberg/util`? @pedroerp